pkg/scheduler/reconcile_on_edgeplacement.go

```diff
@@ -83,7 +83,7 @@ func (c *controller) reconcileOnEdgePlacement(ctx context.Context, epKey string)
 
 	/*
 		On EdgePlacement change:
-		- find all its loc(s)
+		- find all its loc(s), and update store --- how to get its previous loc(s)?
```

Comment:
The store would be the natural place to get the previous set of matching locations